frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-broker-trigger.feature

```diff
@@ -31,6 +18,19 @@ Feature: Knative Eventing Broker and Trigger
              Then Subscriber has knative service dropdown with "Service" and "KSVC" options
 
 
+        @smoke 
+        Scenario: Add Trigger to Broker: KE-04-TC02
+            Given user is having Broker "broker-test1" on the Topology page
+              And user is at Topology page
```

Comment:
Mention subscriber in the given statement:
user has created knative service "nodejs-ex-git"